app/inline-scripts/load-svg.js

```diff
@@ -4,26 +4,21 @@
 
 		template.insertAdjacentHTML('beforeend', html);
 
-		return template.firstChild;
+		return template.querySelector('svg');
 	}
 
 	function loadDOMResource(src) {
-		var ajax = new XMLHttpRequest();
+		fetch(src)
+			.then(function(response) {
+				return response.text();
```

Comment:
What happens when it throws? You can check it by blocking svg network request in DevTools.